TypeScript.YAML-tmLanguage

```diff
@@ -140,7 +140,7 @@ variables:
     {{functionLikeType}} |
     (:\s*(=>|{{matchingParenthesis}}|(<[^<>]*>)|[^<>(),=])+={{functionOrArrowLookup}})
   arrowFunctionEnd: (?==>|\{|(^\s*(export|function|class|interface|let|var|{{usingKeyword}}|{{awaitUsingKeyword}}|const|import|enum|namespace|module|type|abstract|declare)\s+))
-  regexpTail: ([dgimsuy]+|(?![\/\*])|(?=\/\*))(?!\s*[a-zA-Z0-9_$])
+  regexpTail: (?!\*)[dgimsuy]*((?!\/)|(?=\/\*))(?!\s*[a-zA-Z0-9_$])
```

Comment:
Are you sure this shouldn't also be applied to other tmLanguage files as done in #1032?

The PR also seems to be missing updates to the tmLanguage files themselves.

Reply:
is there no auto github workflow?

Reply:
I don't think so, you should just be able to run `npm run build:grammar`.

It'd be good if CI verified that this had been run.